### PR TITLE
163451566 detected changed help text

### PR DESCRIPTION
**Pull request by rckopteri**
# Changed
* transit changes page: detected changed help text
* Few text updates to "mahdollisesti päättyvät reitit" category
* small ui element updates

### CODE REVIEW

File: ote/src/cljs/ote/style/base.cljs [hunk 1]
```diff
@@ -207,7 +207,7 @@
    :align-items "center"})
 
 (def icon-labeled-icon
-  {:margin-right "0.25rem"})
+  {:margin-right "0.125rem"})
```
Comment (philosotractor) [inline ote/src/cljs/ote/style/base.cljs]: Just double checking, was it checked that all places where this affects look good? 
e.g. success_msg and warning_msg.

File: ote/src/cljs/ote/style/transit_changes.cljs [hunk 2]
```diff
@@ -6,7 +6,7 @@
 
 (def add-color "rgb(0,170,0)")
 (def remove-color "rgb(221,0,0)")
-(def no-change-color "rgb(235,235,235)")
+(def no-change-color colors/gray700)
```
Comment (philosotractor) [inline ote/src/cljs/ote/style/transit_changes.cljs]: 👍

File: ote/src/cljs/ote/views/transit_visualization.cljs [hunk 3]
```diff
@@ -405,7 +406,11 @@
                     :removed
                     [icon-l/icon-labeled
                      [ic/content-remove-circle-outline {:color style/remove-color}]
-                     "Mahdollisesti päättyvä reitti"]
+                     [:span {:title (str "Reitti päättyy mahdollisesti "
+                                         (time/format-timestamp->date-for-ui different-week-date)
+                                         ". "
+                                         "Ota liikennöitsijään yhteyttä saadaksesi tarkempaa informaatiota.")}
```
Comment (philosotractor) [inline ote/src/cljs/ote/views/transit_visualization.cljs]: "tarkempia tietoja." perhaps?

File: ote/src/cljs/ote/style/base.cljs (hunk 1 above)
Comment (rckopteri) [reply ote/src/cljs/ote/style/base.cljs]: Change is small and I think that it won't make success messages to look bad.
But I didn't test it. 
But it looks good in transit visualization page even when icons has trailing text (<icon> Mahollisesti päättyvä reitti)

File: ote/src/cljs/ote/ui/page.cljs [hunk 4]
```diff
@@ -17,7 +17,7 @@
   the rest of the page."
   [top-link title content]
   [:div.page-controls (stylefy/use-style style/page-controls)
-   [:div.container
+   [:div.container (when (not (empty? top-link)) {:style {:padding-top "1rem"}})
```
Comment (mattiuusitalo) [inline ote/src/cljs/ote/ui/page.cljs]: when-not

**Pull request merged**